src/transform-middleware.ts

```diff
@@ -25,7 +25,9 @@ export function transformResponse(transformer: ResponseTransformer):
 
     function shouldTransform() {
```

Comment:
For clarity, since even I forgot I did this, can you mention that this function memoizes it's result